protocol/v2/ssv/runner/committee.go

```diff
@@ -237,6 +241,12 @@ func (cr *CommitteeRunner) ProcessConsensus(ctx context.Context, logger *zap.Log
 		switch validatorDuty.Type {
 		case spectypes.BNRoleAttester:
 			validDuties++
+
+			if cr.doppelgangerHandler.ValidatorStatus(validatorDuty.ValidatorIndex) != doppelganger.SigningEnabled {
+				logger.Warn("Doppelganger check in progress, signing not permitted", fields.ValidatorIndex(validatorDuty.ValidatorIndex))
```

Comment:
what about a case where there's only one valid duty but it's a doppleganger duty. might make us send an empty post-consensus message.

Reply:
good catch!